docs/security/apparmor.md

```diff
@@ -0,0 +1,87 @@
+AppArmor Policies for Docker
+----------------------------
+
+This directory contains two files and three policies for Docker:
+
+1. File `docker` for container policies
+  * docker-default
+  * docker-unconfined
+2. File `docker-engine` for the engine policy
+  * /usr/bin/docker
+
+These policies are installed by the .deb packages for
+Debian-derived distributions, including Ubuntu.
+
+Other packages are free to incorporate these policies
+and operating system vendors are encouraged to install
+policies with Docker.
+
+Installing Policies
+-------------------
+
+Policies should normally be installed through your
+operating system packages. However, those developing
+Docker, building their own packages, or simply desiring
+to install these policies manually may load them using
+`apparmor_parser`.
+
+For example:
+
+```
+$ sudo apparmor_parser -r contrib/apparmor
```

Comment:
Yes, this is loading a policy.

The difference between loading and installing is that installing is persisting across reboots.

Applying a policy is *using* a policy that has been loaded.